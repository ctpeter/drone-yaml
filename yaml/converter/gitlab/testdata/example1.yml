image: ruby:2.2

get_env_var_from_vault_sg_staging:
  - postgres:9.3

before_script:
  - bundle install

test:
  script:
  - bundle exec rake spec
