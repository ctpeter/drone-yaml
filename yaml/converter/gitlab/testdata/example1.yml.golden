---
kind: pipeline
name: test

platform:
  os: linux
  arch: amd64

steps:
- name: test
  image: ruby:2.2
  commands:
  - bundle install
  - bundle exec rake spec

get_env_var_from_vault_sg_staging:
- name: postgres
  image: postgres:9.3

...
